File: SDKs/Aspose.Pdf_Cloud_SDK_for_.NET/src/Com/Aspose/PDF/Model/ResponseMessage.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.PDF.Model {
    public class ResponseMessage
    {
        public ResponseMessage() { }
        public ResponseMessage(byte[] ResponseStream, int Code = 200, String Status = "Ok")
        {
            this.Code = Code;
            this.Status = Status;
            this.ResponseStream = ResponseStream;
        }
        public int Code { get; set; }
        public string Status { get; set; }

        public string Message { get; set; }
        public byte[] ResponseStream { get; set; }

        public override string ToString()
        {
            var sb = new StringBuilder();
            sb.Append("class ResponseMessage {\n");
            sb.Append("  Code: ").Append(Code).Append("\n");
            sb.Append("  Message: ").Append(Message).Append("\n");
            sb.Append("}\n");
            return sb.ToString();
        }
    }
  }

File: SDKs/Aspose.Pdf-Cloud-SDK-for-Android/Aspose.Storage-Cloud-SDK-for-Android/src/main/java/com/aspose/storage/model/ResponseMessage.java
package com.aspose.storage.model;

import java.io.InputStream;

/**
 *
 * @author Imran Anwar
 * This class is added for the response to be separated in Json and InputStream
 */
public class ResponseMessage {
  private Integer Code = null;
  private String Message = null;
  private String Status = null;
  private InputStream inputStream = null;
  public InputStream getInputStream() {
    return inputStream;
  }
  public void setInputStream(InputStream inputStream) {
    this.inputStream = inputStream;
  }
  public Integer getCode() {
    return Code;
  }
  public void setCode(Integer Code) {
    this.Code = Code;
  }
  public String getMessage() {
    return Message;
  }
  public void setMessage(String Message) {
    this.Message = Message;
  }
  public String getStatus() {
    return Status;
  }
  public void setStatus(String Status) {
    this.Status = Status;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class ResponseMessage {\n");
    sb.append("  Code: ").append(Code).append("\n");
	sb.append("  Status: ").append(Status).append("\n");
    sb.append("  Message: ").append(Message).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}


